.env.example

```diff
@@ -1,4 +0,0 @@
-DISCORD_HASH=
-GITHUB_TOKEN=
-AMPLITUDE_KEY=
-BOT_LOG=

```

Comment:
We still need this file to help contributors set up the bot locally

Reply:
Why ?? You could just do same with real .env file. Just remove keys simple